akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -309,8 +312,9 @@ final class ORMap[A, B <: ReplicatedData] private[akka] (
    * by keeping the vvector (in form of key -> value pair) for deleted keys
    */
   @InternalApi private[akka] def removeKey(node: UniqueAddress, key: A): ORMap[A, B] = {
-    val removeKeyDeltaOp = RemoveKeyDeltaOp(keys.resetDelta.remove(node, key).delta.get, key, zeroTag)
-    new ORMap(keys.remove(node, key), values, zeroTag, Some(newDelta(removeKeyDeltaOp)))
+    val newKeys = keys.resetDelta.remove(node, key)
```

Comment:
ok, so the fix is to always `resetDelta` of the keys.
Does it make sense to simulate the issue in a unit test (`ORMapSpec`)?